src/main/scala/com.snowplowanalytics/iglu.schemaddl/migrations/FlatSchema.scala

```diff
@@ -20,15 +20,18 @@ import cats.instances.list._
 import cats.instances.option._
 import cats.syntax.alternative._
 
-import com.snowplowanalytics.iglu.schemaddl.{ SubSchemas, StringUtils }
+import io.circe.Json
+
+import org.json4s.jackson.JsonMethods.compact
```

Comment:
Is this the last thing from json4s namespace?

Reply:
Just for FlatSchema ?

Reply:
It seems that for the whole Schema DDL (apart from type class instances obviously)

Reply:
I didn't look at it in detailed but most probably it is as you said. I will try to remove this usage of it 👍